src/client/utils/helpers.ts

```diff
@@ -294,3 +294,10 @@ export const getNoteBarConf = (
 export const copyToClipboard = (text: string) => {
   clipboard.writeText(text)
 }
+
+export const unboldNote = (note: string) => {
+  var unboldedText = note.replace(/\*\*([^*]*(?:\*(?!\*)[^*]*)*)\*\*/g, function (match, index) {
```

Comment:
fairly sure this regex also cleans up stuff **that is not ** bold :) https://www.markdownguide.org/basic-syntax/